core/views.py

```diff
@@ -36,3 +37,30 @@ def gh_webhook(request):
         return HttpResponse(handle_webhook(request))
     except KeyError:
         return HttpResponse("Invalid request", status=500)
+  
+
+def handling_404(request, exception):
+    context = {
+        "error_id": "404",
+        "description": "Ojej! Wygląda na to, że jesteś teraz na drodze do nikąd :o",
+        "image": static("images/404_image.png"),
+    }
+    return render(request, "core/error.html", context, status=404)
+
+
+def handling_500(request):
+    context = {
+        "error_id": "500",
+        "description": "Ojej, coś jest nie tak z naszym serwerem :(",
+        "image": static("images/500_image.png"),
+    }
+    return render(request, "core/error.html", context, status=500)
+
+
+def handling_403(request, exception):
```

Comment:
Why sometimes functions have `request, exception`, but sometimes just `reqest`?

Reply:
when i put only request in handling 403 there was an error. so should i just put both req and excp in all of the functions?